src/components/message/Message.tsx

```diff
@@ -39,7 +47,7 @@ const Message: FC<MessageProps> = ({ message }) => {
     <Link onClick={handleLinkClick} to={userURL}>
       <Avatar
         src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
-        sx={styles.avatar}
+        sx={spliceSx(styles.avatar, sx.avatar)}
```

Comment:
If you are passing sx only to avatar, maybe better set avatar styles to sx

Reply:
I guess that we will change styles for the mobile version and need to pass styles for text, so I decided don't change it.